### DISCUSSION TITLE
How to configure proxy for RSS feeds on netbox

### DISCUSSION THREAD

**Discussion by el-magneto** [netbox-community/netbox#15504] [Q&A]
Hi!

Is it possible to configre netbox to use proxy when connecting to Internet websites? My netbox is behind proxy and RSS Feed Configuration doesn't have options to configre proxy so I get an error.

Thanks in advance!

**Comment by candlerb** [comment]
> Is it possible to configre netbox to use proxy when connecting to Internet websites?

Yes.

https://docs.netbox.dev/en/stable/configuration/system/#http_proxies

**Reply by el-magneto** [reply]
Thank you! It works for http feeds. But I have problem with https feeds. Reason: CERTIFICTE_VERIFY_FAILED. It looks like my proxy cert is not recognised. So two questions:
1. Is it an option in netbox to disable certificate veryfication? I couldn't find it in netbox documentation.
2. If I want to add CA cert to trusted certs should I do that on OS level? Or on some netbox setting/store? I'm using Ubuntu.
